bedrock/firefox/templates/firefox/whatsnew/whatsnew-rocket-id.html

```diff
@@ -0,0 +1,57 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew" %}
+
+{% extends "firefox/whatsnew/base-pebbles.html" %}
+
+{% block page_title %}Firefox kamu sudah versi terbaru.{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  Mau coba Android browser terbaru yang hemat data, ringan, secepat kilat, dan dibuat khusus untuk Indonesia?
+{% endblock %}
+
+{% block body_id %}firefox-mobile-download{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox-whatsnew-rocket' %}
+{% endblock %}
+
+
+{% block content %}
+<main role="main" class="section">
+  <div class="content">
+    <a class="moz-logo" href="{{ url('mozorg.home') }}"><img src="{{ static('img/logos/mozilla/wordmark-dark.svg')}}" alt="Mozilla" width="101" height="34"></a>
+
+    <div class="rocket-content">
+      <div class="content-intro">
+        {# Rough translation: "You have the latest version of Firefox" #}
+        <h1>Firefox kamu sudah versi terbaru.</h1>
+
+        {# Rough translation: "Want to try the latest Android browser that is data-efficient, lightweight, lightning fast, and tailor-made for Indonesia?" #}
+        <p>Mau coba Android browser terbaru yang hemat data, ringan, secepat kilat, dan dibuat khusus untuk Indonesia?</p>
+
+        {{ high_res_img('logos/firefox/logo-rocket-wordmark.png', {'alt': 'Firefox Rocket', 'width': '470', 'height': '112', 'class': 'rocket-logo'}) }}
+      </div>
+
+      <figure class="qr-code">
+        <img src="{{ static('img/firefox/whatsnew/rocket/qrcode-rocket.png') }}" alt="">
+
+        {# Rough translation: "Scan the QR code and download the new Firefox Rocket app now!" #}
+        <figcaption>Scan kode QR dan download aplikasi baru Firefox Rocket sekarang!</figcaption>
+      </figure>
+    </div>
+  </div>
+</main>
+{% endblock %}
+
+{% block email_form %}{% endblock %}
+
+{% block js %}
+  {% javascript 'firefox_whatsnew' %}
```

Comment:
Is this JS bundle needed?

Reply:
oh, probably not... just carried over from the template I copied 😁